### DISCUSSION TITLE
Trouble connecting to the server with Thunder Client via VS Code on my MacOS

### DISCUSSION THREAD

**Discussion by remoteconn-7891** [community/community#75324] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

So I'm trying to connect to the server using Thunder Client on VS Code. I'm primarily using express.js in conjunction with MongoDB for the backend for my project. The problem is that I keep getting an error 'Connection was refused by the server' with ThunderClient. I've tried with ChatGPT with no success, the Firewall is disabled on my MacOS, the PORT on my entry point file, index.js is correct, since it's using PORT 3000 and is connected to MongoDB. I believe the code for express.js is correct for all server files index.js, auth.js and user.js. I'll copy/paste a snippet of index.js. Please I've been stuck on this issue since yesterday. I even tried stack overflow but got blocked from asking anymore questions and ChatGPT wasn't helpful. Please let me know if you need anything else from me including screenshots or code snippets from the other express files

`const express = require("express");
const cors = require("cors");
const mongoose = require("mongoose");
const dotenv = require("dotenv"); // Import dotenv 
const authRouter = require("./routes/auth");
dotenv.config(); // Load environment variables // Load environment variables from .env file
const PORT = process.env.PORT || 3000;
const app = express();

app.use(express.json());
app.use(authRouter);
//DB just stands for MongoDB database, this String is the password for my database
const DB = 
"mongodb+srv://coreyjames689:remotify87@cluster0.jybj21u.mongodb.net/Cluster0?retryWrites=true&w=majority";

mongoose 
  .connect(DB)
  .then(() => {
    console.log("Connection Successful");
  })
  .catch((e) => {
    console.log(e);
  });

  //Using PORT 3000 for Node.JS
  app.listen(3000, "0.0.0.0", () => {
    console.log(`Server is running on port 30000`);
  });`

**Comment by rangav** [comment]
Hi @remoteconn-7891 please try suggestions in this link https://github.com/rangav/thunder-client-support/blob/master/docs/common-issues.md

**Comment by remoteconn-7891** [comment]
I'm currently watching a tutorial video on creating a custom login screen. In the video, he uses PORT 3000 in the URL for POST. I'll try this though

**Comment by remoteconn-7891** [comment]
Either method doesn't seem to be working

**Comment by remoteconn-7891** [comment]
<img width="1440" alt="Screenshot 2023-11-13 at 12 18 17 AM" src="https://github.com/community/community/assets/142646120/af370359-973b-4ad0-a413-a3c73c0d4077">
